standard/classes.md

```diff
@@ -442,8 +442,6 @@ The value type constraint specifies that a type argument used for the type param
 
 > *Note*: The `System.Nullable<T>` type specifies the non-nullable value type constraint for `T`. Thus, recursively constructed types of the forms `T??` and `Nullable<Nullable<T>>` are prohibited. *end note*
 
-Pointer types are never allowed to be type arguments and are not considered to satisfy either the reference type or value type constraints.
```

Comment:
I think we're OK, but I'll defer to others. My thinking is that a pointer type has an explicit conversion to the integral types, but only has implicit conversions to `void*`. That's also a pointer type, so any option for type inference would be a pointer type, and would fail.